docs/snippets/install-kn.md

```diff
@@ -1,3 +1,5 @@
+## Install the Knative CLI
```

Comment:
Could you add a comment for where this snippet is found like in you did for the other snippets. Adding this info to snippets should probably be our standard going forward.

This snippet is also in /docs/install/client/install-kn.md